docs/howto/write-logs.rst

```diff
@@ -72,36 +72,19 @@ In the above example, Airflow will try to use ``S3Hook('MyS3Conn')``.
 Writing Logs to Azure Blob Storage
 ----------------------------------
 
-Airflow can be configured to read and write task logs in Azure Blob Storage.
-
 Follow the steps below to enable Azure Blob Storage logging:
 
-#. Airflow's logging system requires a custom ``.py`` file to be located in the ``PYTHONPATH``, so that it's importable from Airflow. Start by creating a directory to store the config file, ``$AIRFLOW_HOME/config`` is recommended.
-#. Create empty files called ``$AIRFLOW_HOME/config/log_config.py`` and ``$AIRFLOW_HOME/config/__init__.py``.
-#. Copy the contents of ``airflow/config_templates/airflow_local_settings.py`` into the ``log_config.py`` file created in ``Step 2``.
-#. Customize the following portions of the template:
-
-    .. code-block:: ini
-
-        # wasb buckets should start with "wasb" just to help Airflow select correct handler
-        REMOTE_BASE_LOG_FOLDER = 'wasb-<whatever you want here>'
-
-        # Rename DEFAULT_LOGGING_CONFIG to LOGGING CONFIG
-        LOGGING_CONFIG = ...
-
-
-#. Make sure a Azure Blob Storage (Wasb) connection hook has been defined in Airflow. The hook should have read and write access to the Azure Blob Storage bucket defined above in ``REMOTE_BASE_LOG_FOLDER``.
-
-#. Update ``$AIRFLOW_HOME/airflow.cfg`` to contain:
-
-    .. code-block:: bash
+.. code-block:: ini
 
-        remote_logging = True
-        logging_config_class = log_config.LOGGING_CONFIG
```

Comment:
This is a very valuable piece of documentation, although it may not be well written. This contains the description of the `logging_config_class` key.

Reply:
Maybe, this description can be move to advanced configuration section.

Reply:
I have moved the description of the `logging_config_class key` to Customize Logging Configuration section.